src/bootstrap/config.rs

```diff
@@ -1129,6 +1130,17 @@ impl Config {
         };
 
         if let Some(include) = &toml.profile {
+            // Allows creating alias for profile names, allowing
+            // profiles to be renamed while maintaining back compatibility
+            // Keep in sync with `profile_aliases` in bootstrap.py
+            static PROFILE_ALIASES: OnceLock<HashMap<&str, &str>> = OnceLock::new();
+            let include = match PROFILE_ALIASES
+                .get_or_init(|| [("user", "dist")].into_iter().collect())
+                .get(include.as_str())
+            {
+                Some(alias) => alias,
+                None => include.as_str(),
+            };
```

Comment:
i wouldn't bother with the static, bootstrap isn't perf-sensitive.
```suggestion
            let profile_aliases = HashMap::from([("user", "dist")]);
            let include = profile_aliases.get(include.as_str()).unwrap_or(include.as_str())
```